tests/StorageApiIndexClientTest.php

```diff
@@ -24,7 +24,7 @@ class StorageApiIndexClientTest extends TestCase
     public function testFunctional(): void
     {
         $client = new StorageApiIndexClient(
-            (string) getenv('STORAGE_API_URL'),
+            sprintf('https://connection.%s', (string) getenv('HOSTNAME_SUFFIX')),
```

Comment:
Zatim takhle. Service klienta jsem nechtel pridavat kdyz je to jen na jednom miste.
Navic bych musel updatnout i min PHP na 8.x a predelat Docker image.